### PR TITLE
Announce 2020-october coderefinery

### PR DESCRIPTION
**Pull request by rkdarst**
May need some work.  announce stream now or later?

### CODE REVIEW

File: tweets/2020-10-coderefinery-announce.tweet [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Computational researcher?  Program, but not a programmer? Join CodeRefinery to learn best sofwware practices for research
```
Comment (bast) [inline tweets/2020-10-coderefinery-announce.tweet:1]:
```suggestion
Are you a computational researcher? Are you writing code, but you are not a programmer? Join CodeRefinery to learn best software practices for research.
```

**Comment by rkdarst** [comment]
i accepted your proposal, but because of the settings someone else needs to review again before we can send it.

**Comment by naoe-tatara** [comment]
I think the tweet looks good.

**Comment by rkdarst** [comment]
We need someone to approve this (Files changed -> Review changes -> Approve), but maybe it's best to do it Monday.

**Review by annefou — APPROVED** [review]

**Comment by rkdarst** [comment]
Has this already been tweeted some other way, or should it be sent from here?

**Comment by bast** [comment]
I don't think it has been. Merging.

**Pull request merged**